crates/compilers/src/compilers/resolc/compiler.rs

```diff
@@ -35,7 +35,17 @@ impl Compiler for Resolc {
     type Language = SolcLanguage;
 
     fn compiler_version(&self, _input: &Self::Input) -> Version {
-        self.resolc_version.clone()
+        let mut v = self.resolc_version.clone();
```

Comment:
If I understand this correctly, now we combine resolc with solc version, so why in hasher we need to hash input and compiler_version?

Reply:
hmm ,good call. I dunno what to do  with `short_version` in the build_info hash tho, for removal as well as it might not work properly?